frontend/src/pages/Offer.jsx

```
@@ -0,0 +1,183 @@
+    Accepted: <CheckCircle className="text-emerald-400" />,
+    Rejected: <XCircle className="text-rose-400" />
+  };
+
+  const resetFilters = () => {
+    setSearchQuery("");
+    setFilterStatus("All");
+  };
+
+  if (loading) return (
+    <div className="flex justify-center items-center h-screen">
+      <div className="animate-spin w-16 h-16 border-4 border-blue-500 border-t-transparent rounded-full"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)